app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt

```diff
@@ -60,18 +69,31 @@ import com.example.bluromatic.ui.theme.BluromaticTheme
 @Composable
 fun BluromaticScreen(blurViewModel: BlurViewModel = viewModel(factory = BlurViewModel.Factory)) {
     val uiState by blurViewModel.blurUiState.collectAsStateWithLifecycle()
-    BluromaticTheme {
-        Surface(
-            modifier = Modifier.fillMaxSize(),
-            color = MaterialTheme.colorScheme.background
-        ) {
-            BluromaticScreenContent(
-                blurUiState = uiState,
-                blurAmountOptions = blurViewModel.blurAmount,
-                applyBlur = blurViewModel::applyBlur,
-                cancelWork = {}
+    val layoutDirection = LocalLayoutDirection.current
+    Surface(
+        modifier = Modifier
+            .fillMaxSize()
+            .statusBarsPadding()
+            .padding(
+                start = WindowInsets.safeDrawing.asPaddingValues()
+                    .calculateStartPadding(layoutDirection),
+                end = WindowInsets.safeDrawing.asPaddingValues()
+                    .calculateEndPadding(layoutDirection)
             )
-        }
+    ) {
+        BluromaticScreenContent(
+            blurUiState = uiState,
+            blurAmountOptions = blurViewModel.blurAmount,
+            applyBlur = blurViewModel::applyBlur,
+            cancelWork = {},
+            modifier = Modifier
+                .verticalScroll(rememberScrollState())
+                .padding(
+                    start = dimensionResource(R.dimen.padding_small),
+                    top = dimensionResource(R.dimen.padding_small),
+                    end = dimensionResource(R.dimen.padding_small),
+                )
```

Comment:
Could we change this to 

`.padding(dimensionResource(R.dimen.padding_medium))`

Where padding_medium is 16dp

`    <dimen name="padding_medium">16.dp</dimen>
`

Reply:
Agreed. Done.